### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *gorgeous-sawine-f42360* failed.

### ISSUE THREAD

**Issue by kyleeelmoo** [kyleeelmoo/Buildingwise-by-kyle#2]
### <span aria-hidden="true">❌</span> Deploy Preview for *gorgeous-sawine-f42360* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 277de412265b5710d6f1ac50d469acd3b37c909e |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/gorgeous-sawine-f42360/deploys/68f547723ce122000734f72c |

_Originally posted by @netlify in https://github.com/kyleeelmoo/superintendent-personal-tracking--app/pull/4#issuecomment-3419928850_
